src/components/ImageView/index.native.js

```diff
@@ -99,20 +106,6 @@ class ImageView extends PureComponent {
     configureImageZoom({nativeEvent}) {
         // Wait till animations are over to prevent stutter in navigation animation
         this.state.interactionPromise = InteractionManager.runAfterInteractions(() => {
-            this.setState({imageHeight: 0, imageWidth: 0, isLoading: true});
-            if (this.imageZoomScale !== 1) {
-                this.imageZoomScale = 1;
-            }
-
-            if (this.zoom) {
-                this.zoom.centerOn({
-                    x: 0,
-                    y: 0,
-                    scale: 1,
-                    duration: 0,
-                });
-            }
-
```

Comment:
Shouldn't we call reset zoom, here too?

Reply:
It's too late in the image lifecycle because here it's loaded and you get this weird animation on loaded where the new image zooms out

Reply:
Oh, I got it!